blackfootwords/datatables.py

```diff
@@ -1,9 +1,24 @@
 from sqlalchemy.orm import joinedload
 from clld.web import datatables
 from clld.web.datatables.base import LinkCol, Col, LinkToMapCol
+from clld.web.datatables.value import Values
+from clld.db.util import get_distinct_values
 
 from blackfootwords import models
 
 
+class Lemmas(Values):
+    def col_defs(self):
+        res = Values.col_defs(self)
```

Comment:
We want to add a column, so we override the method specifying the list of columns. Since we want to keep the default columns as well, we first call the default implementation ...